### DISCUSSION TITLE
Use getter setter in classes in bmp.h

### DISCUSSION THREAD

**Discussion by Ank1taS** [Blaze-Stars/Image-Filter-Application#22] [Q&A]
Classes of bmp.h contain all public attributes. As we implement C++, it is better to keep all attributes private and have public getters and setters.

revert back to https://github.com/Blaze-Stars/Image-Filter-Beta/pull/8

**Comment by H1manshu21** [comment]
Getters and Setters should be used for `classes` in `bmp.h` so that it will follow the OOPs paradigm. @Ank1taS @chandrakant100 @HembramBeta777 @sabyasachi07
